content/BindingMethods/BindingMethods.objectives.md

```diff
@@ -0,0 +1,11 @@
+### Objectives for Prepared Statement with Binding Methods
+
+After completing the lesson, the students should be able to:
+
+- Identify the different binding methods.
+
+- Distinguish the difference between `PDOStatement::bindParam()` and `PDOStatement::bindValue()`.
+
+- Use `PDOStatement::bindParam()` and `PDOStatement::bindValue()` in binding placeholders as parameters in an SQL statement.
```

Comment:
Is it the placeholders that the binding methods need to bind? Please check. Thanks.